pkg/pipelines/build.go

```diff
@@ -32,6 +33,10 @@ func BuildResources(o *BuildParameters, appFs afero.Fs) error {
 		return err
 	}
 	_, err = yaml.WriteResources(appFs, o.OutputPath, resources)
+
+	if err == nil {
```

Comment:
Generally the pattern is `if err != nil { ... return err }` otherwise do the successful thing.

Reply:
Also, I'm not sure we need all the bangs :-)